src/components/PostPageDetail/index.js

```diff
@@ -68,11 +77,12 @@ const PostPageDetailIdComponent = (props) => {
   const {getTotalReaction, getHeightHeader} = useFeed();
   const [commentContext, dispatchComment] = React.useContext(Context).comments;
   const {comments} = commentContext;
-  const [, setLoadingGetComment] = React.useState(true);
+  const [loadingGetComment, setLoadingGetComment] = React.useState(true);
   const {updateVoteLatestChildrenLevel3, updateVoteChildrenLevel1, calculatePaddingBtm} =
     usePostDetail();
   const {updateFeedContext} = usePostContextHook(contextSource);
   const {updateFeedContext: updateTopicContext} = usePostContextHook(CONTEXT_SOURCE.TOPIC_FEEDS);
+  const {width: displayWidth} = useWindowDimensions();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new state variable `loadingGetComment` has been introduced to track the loading state of the comment fetching process. Also, `useWindowDimensions` hook is used to get the display width. These changes seem appropriate for the intended functionality.
```diff
- const [, setLoadingGetComment] = React.useState(true);
+ const [loadingGetComment, setLoadingGetComment] = React.useState(true);
+ const {width: displayWidth} = useWindowDimensions();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->